rpm-x64/Dockerfile

```diff
@@ -255,7 +255,9 @@ RUN curl -sSL -o rustup-init "https://static.rust-lang.org/rustup/archive/${RUST
     && chmod +x ./rustup-init \
     && ./rustup-init -y --profile minimal --default-toolchain "${RUST_VERSION}" \
     && echo "${RUSTC_SHA256}  $HOME/.cargo/bin/rustc" | sha256sum --check \
-    && rm ./rustup-init
+    && rm ./rustup-init \
+    # We need this so that we can target musl instead of (a very old) glibc
+    && rustup target add x86_64-unknown-linux-musl
```

Comment:
You're correct 👍. I'm trying adding the `-t x86_64-unknown-linux-musl` to `rustup-init` instead (https://github.com/rust-lang/rustup/blob/87fa15d13e3778733d5d66058e5de4309c27317b/rustup-init.sh#L56-L57). I suspect that otherwise just `PATH="~/.cargo/bin:${PATH}" rustup target add x86_64-unknown-linux-musl` might work as well without adding the extra docker layer.